### PR TITLE
[oneDNN] Fix _FusedConv2D crash in oneDNN enabled build

### PR DESCRIPTION
**Pull request by yimeisun123**
A previous commit 5be9a59 changes op definition of _FusedConv2D, but this causes a side effect of crashes when executing _FusedConv2D during run time in oneDNN enabled build. This PR provides quick fix for the crash.

### CODE REVIEW

**Comment by yimeisun123** [comment]
@penpornk, this is a workaround for the run time execution of _FusedConv2D crash in oneDNN enabled build. Thanks

File: tensorflow/core/grappler/optimizers/remapper.cc [hunk 1]
```diff
@@ -2635,6 +2635,11 @@ Status AddFusedContractionNode(
   const NodeDef& bias_add = graph->node(matched.bias_add);
   fused_conv.add_input(bias_add.input(matched.bias_port));  // 2: bias
 
+  // Add OP has two inputs, one is conv+bias pattern matched previously,
+  // the other input to add is fused here.
+  const NodeDef& add = graph->node(matched.add);
+  fused_conv.add_input(add.input(1 - matched.port_id));
```
Comment (penpornk) [inline tensorflow/core/grappler/optimizers/remapper.cc:2641]: Could you please explain why we need to change the order here?

**Review by penpornk — APPROVED** [review]
Thank you for the quick workaround!

File: tensorflow/core/grappler/optimizers/remapper.cc (hunk 1 above)
Comment (yimeisun123) [reply tensorflow/core/grappler/optimizers/remapper.cc:2641]: I need to move the add_input to before calling the CopyConv2DAttributes due to the change from the commit 5be9a59, specifically, the following line requires all the add_input has been called to get the correct input_size. 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/grappler/optimizers/remapper.cc#L2139

Thanks for the quick review.

**Pull request merged**